From: Andreas Stieger <Andreas.Stieger@gmx.de>
Date: Sat, 19 Apr 2025 07:40:16 +0000
Subject: [PATCH] fix build with cmake 4
References: https://bugzilla.opensuse.org/show_bug.cgi?id=1239788

This is part of an upstream change that fixed the build with cmake 4 as
a side-effect.

https://github.com/rpm-software-management/createrepo_c/pull/438
https://github.com/rpm-software-management/createrepo_c/commit/89fa02828cdaf1c710c38bde5fcbcf59538a9cce

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9a18393f..c4bf5252 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-CMAKE_MINIMUM_REQUIRED (VERSION 2.8.12)
+CMAKE_MINIMUM_REQUIRED (VERSION 3.7)
 PROJECT (createrepo_c C)
 
 include(GNUInstallDirs)
